packages/react-component-library/src/components/Toast/partials/StyledToast.tsx

```diff
@@ -10,6 +10,17 @@ interface StyledToastProps {
 
 const { shadow, color, spacing } = selectors
 
+function getColor(appearance: AppearanceTypes): string {
```

Comment:
You could also just have:

```
  const colors = {
    success: color('success', '500'),
    error: color('danger', '500'),
    warning: color('warning', '500'),
    info: color('action', '500'),
  }
```

Rather than the `function` and then:

```
color: ${colors[$appearance]};
```